Testreglar/3.3.1/App/app-3.3.1a-2022.json

```diff
@@ -0,0 +1,172 @@
+            "type": "jaNei",
+            "ruting": {
+                "ja": {
+                    "type": "ikkjeForekomst",
+                    "utfall": "Skjema uten automatisk oppdaging av feil.",
+                    "element": "2.3"
+                },
+                "nei": {
+                    "type": "gaaTil",
+                    "steg": "3.1"
+                }
+            }
+        },
+        {
+            "stegnr": "3.1",
+            "spm": "Beskriv skjemaelementet som skal vurderast.",
+            "ht": "Beskriv skjemaelementet du skal teste. Dersom det finst skjemaelement i skjembilde må du registrere dei kvar for seg.",
+            "type": "tekst",
+            "label": "Skjemaelement:",
+            "multilinje": true,
+            "oblig": true,
+            "ruting": {
+                "alle": {
+                    "type": "gaaTil",
+                    "steg": "3.2"
+                }
+            }
+        },
+        {
+            "stegnr": "3.2",
+            "spm": "Blir feilmeldinga lest opp med skjermlesar?",
+            "ht": "<p>Aktiver skjermlesar (opplesingsfunksjon). Trykk på feilmeldinga. Blir feilmeldinga lest opp med skjermlesaren?</p>",
+            "type": "jaNei",
+            "kilde": [
+                "G83"
+            ],
+            "ruting": {
+                "ja": {
+                    "type": "gaaTil",
+                    "steg": "3.3"
+                },
+                "nei": {
+                    "type": "avslutt",
+                    "fasit": "Nei",
+                    "utfall": "Obligatoriske skjemaelement der feil utfylling vert oppdaga automatisk, gir feilmelding som ikkje blir lest opp med skjermlesar."
+                }
+            }
+        },
+        {
+            "stegnr": "3.3",
+            "spm": "Inneheld feilmeldinga tekst som identifiserer kvar feilen har oppstått?",
+            "ht": "Feilmeldinga må innehalde informasjon som identifiserer skjemaelementet som feilutfylt.",
+            "type": "jaNei",
+            "kilde": [
+                "G83"
+            ],
+            "ruting": {
+                "ja": {
+                    "type": "gaaTil",
+                    "steg": "3.5"
+                },
+                "nei": {
+                    "type": "avslutt",
+                    "fasit": "Nei",
+                    "utfall": "Obligatoriske skjemaelement der feil utfylling vert oppdaga automatisk, gir feilmelding som ikkje identifiserer kvar feilen er oppstått."
+                }
+            }
+        },
+        {
+            "stegnr": "3.5",
+            "spm": "Inneheld feilmeldinga tekst som beskriv feilen?",
+            "ht": "Feilmeldinga gir informasjon om kva feilen består i, slik at brukaren kan finne ut kva som har gått feil, for eksempel \"Fornavn må fyllast ut\". Det er ikkje krav om at feilmeldinga skal innehalde forslag til korleis feil skal rettast. Dette er omfatta av suksesskriterium 3.3.3.",
```

Comment:
samme kommentar som for nett. Skriv om og forenkle to første leddsetninger

Reply:
Vennligst gi tilbakemelding på ny testregel app-3.3.1a.json og ikke på app-3.3.1a-2022.json